tests/test_components.py

```diff
@@ -615,7 +655,28 @@ def test_1d_output_matches_cached_output(self):
         assert True
```

Comment:
You can remove any "assert True" statements, they do nothing.

Reply:
done.